.github/workflows/release-generate-ci.yaml

```diff
@@ -138,6 +138,18 @@ jobs:
           git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/release.git" sync-serverless-ci:sync-serverless-ci -f
           gh pr create --base master --head serverless-qe:sync-serverless-ci --fill-verbose || true
         working-directory: ./src/github.com/openshift-knative/hack/openshift/release
+      - name: Create Pull Request - openshift-knative/hack
+        if: ${{ (github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule') && github.ref_name == 'main' }}
+        env:
+          GH_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
+          GITHUB_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
+        run: |
+          set -x
+          git remote add fork "https://github.com/serverless-qe/hack.git"
+          git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/hack.git" sync-konflux/main:sync-konflux/main -f
```

Comment:
@mgencur we need a fork for the hack repo in `serverless-qe` bot

Reply:
Done. https://github.com/serverless-qe/hack